Directory.Build.props

```diff
@@ -358,9 +358,9 @@
     <TestPath Condition="'$(TestPath)' == ''">$(TestWorkingDir)$(MSBuildProjectName)/$(BuildConfiguration)/</TestPath>
     <RefRootPath>$(BinDir)ref/</RefRootPath>
     <BuildConfigurationRefPath>$(RefRootPath)$(_bc_TargetGroup)/</BuildConfigurationRefPath>
-    <BuildConfigurationRefPath Condition="$(_bc_TargetGroup.EndsWith('aot'))">$(RefRootPath)$(_bc_TargetGroup.TrimEnd('t').TrimEnd('o').TrimEnd('a'))/</BuildConfigurationRefPath>
+    <BuildConfigurationRefPath Condition="$(_bc_TargetGroup.EndsWith('aot'))">$(RefRootPath)$(_bc_TargetGroup.Substring(0, $(_bc_TargetGroup.LastIndexOf('aot'))))/</BuildConfigurationRefPath>
```

Comment:
https://apisof.net/catalog/System.String.TrimEnd(Char()) doesn't work with msbuild?

Reply:
I don't want to trim instances of a, o, or t.  I want to trim the string "aot".  These chained together trimend's and their order was doing that, but were a bit of an eyesore.  I thought this was more readable and avoided too many property function calls (eg: foo.Substring(0, Subtract(foo.length, 3)))